### PR TITLE
Add New Port GUMCHU GIRL

### PR DESCRIPTION
**Pull request by MadShmupper**
Add New Port GUMCHU GIRL

# New Port for { GUMCHU GIRL }

## Game Information
- **Title**: { GUMCHU GIRL }
- **URL**: { caseportman.itch.io/gumchu-girl-demo }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [x] AmberELEC
- [x] ArkOS
- [x] ROCKNIX
- [x] muOS

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [x] 640x480
- [x] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/gumchugirl/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+## Notes
+
+A huge PortMaster thank you goes out graciously to the creator of this game, [CasePortman](https://casePortman.itch.io) for allowing us to distribute this gem.
```
Comment (Cebion) [inline ports/gumchugirl/README.md:3]: Link to the actual game is missing

File: ports/gumchugirl/port.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        "desc_md": null,
+        "inst": "Ready to Run",
+        "inst_md": null,
+        "genres": [
+            "platformer",
+            "puzzle",
+            "strategy"
+        ],
+        "image": {
+            "screenshot": "screenshot.png",
```
Comment (Cebion) [inline ports/gumchugirl/port.json:24]: Why are the image fields filled out?
Did you take a installed port.json file?

**Review by Cebion — CHANGES_REQUESTED** [review]
License file also missing

File: ports/gumchugirl/README.md (hunk 1 above)
Comment (MadShmupper) [reply ports/gumchugirl/README.md]: All should be well now

File: ports/gumchugirl/port.json (hunk 2 above)
Comment (MadShmupper) [reply ports/gumchugirl/port.json]: Fixed it..ty

**Comment by Cebion** [comment]
License.png seems broken

**Review by Cebion — DISMISSED** [review]

File: ports/gumchugirl/GUMCHU GIRL.sh [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+export PORT_32BIT="Y"
+
+get_controls
+[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
+
+$ESUDO chmod 666 /dev/tty0
+
+GAMEDIR="/$directory/ports/gumchugirl"
+
+export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs"
```
Comment (JanTrueno) [inline ports/gumchugirl/GUMCHU GIRL.sh:26]: export LD_LIBRARY_PATH="/usr/lib32:$GAMEDIR/libs:$LD_LIBRARY_PATH"

**Review by JanTrueno — CHANGES_REQUESTED** [review]

File: ports/gumchugirl/GUMCHU GIRL.sh (hunk 3 above)
Comment (MadShmupper) [reply ports/gumchugirl/GUMCHU GIRL.sh:26]: About to hop to it.

**Review by Cebion — APPROVED** [review]

**Review by JanTrueno — APPROVED** [review]

**Pull request merged**